### DISCUSSION TITLE
event param missing from useKeyBindings callback type

### DISCUSSION THREAD

**Discussion by brahambence** [imbhargav5/rooks#609] [Q&A] [answered]
Thank you for creating Rooks, it's pretty awesome!

I started using useKeyBindings in a project, and I need to call preventDefault on the key event in my callback, which TypeScript complained about, since the type declaration for the callback is () => void .

I wonder if this is on purpose and I will somehow shoot myself in the foot for trying to call preventDefault on the event within useKeyBindings hook callback, or if I should create a pull request to add the appropriate optional parameter type?

**Comment by imbhargav5** [comment, marked as answer]
This is most definitely something we didn't really do because we haven't really used it for such a scenario so far. But it is definitely useful and the hook should have it. 

Would you mind doing a PR on it @brahambence ?

**Reply by brahambence** [reply]
Cool, thanks, I created the PR:
https://github.com/imbhargav5/rooks/pull/610
